src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -225,10 +225,10 @@ void activity_classifier::init_table_printer(bool has_validation) {
   }
 }
 
-void activity_classifier::train(gl_sframe data, std::string target_column_name,
-                                std::string session_id_column_name,
+void activity_classifier::train(gl_sframe data, const std::string& target_column_name,
+                                const std::string& session_id_column_name,
                                 variant_type validation_data,
-                                std::map<std::string, flexible_type> opts)
+                                const std::map<std::string, flexible_type>& opts)
```

Comment:
Let's remove the `std::move` usage from these variables. Although they are now harmless, they are misleading.